frontend/src/components/Card.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
```

Comment:
this looks 🔥overall; this component could also use some prop-types 😄 
You can even do fancy stuff like ` type: PropTypes.oneOf(['marketplace', 'collection']),

also, instead of just having strings for 'marketplace' and 'collection', a good paradigm would be do have this component also export a variable with the types, just like how the redux actions do.

E.g. 
```
export const CARD_TYPE_MARKETPLACE = 'marketplace';
export const CARD_TYPE_COLLECTION = 'collection';
```

and then the places that use and then you can do `<CardGrid type={CARD_TYPE_COLLECTION} />`

tl;dr minimizing 'magic' numbers/strings is good!